Automattic/wp-calypso
# Platform: tries swc minifier

## Proposed Changes

This PR adds the `swcMinify` option to the Terser webpack plugin, to enable compilation through SWC

## Testing Instructions

We've tried testing this locally and through TC builds, but couldn't observe a comprehensive reduction in build times. However, we could verify that the builds with the changes still produce the expected minified output, and that the builds contain no breaking changes.

We could release this to production to see if there are real-life benefits with the changes.

## Pre-merge Checklist
